core/dbt/config/profile.py

```diff
@@ -123,10 +126,13 @@ def to_profile_info(self, serialize_credentials: bool = False) -> Dict[str, Any]
             "user_config": self.user_config,
             "threads": self.threads,
             "credentials": self.credentials,
+            "python_adapter_credentials": self.python_adapter_credentials,
         }
         if serialize_credentials:
             result["user_config"] = self.user_config.to_dict(omit_none=True)
             result["credentials"] = self.credentials.to_dict(omit_none=True)
+            if self.python_adapter_credentials is not None:
+                result["python_adapter_credentials"] = self.python_adapter_credentials.to_dict(omit_none=True)
         return result
 
     def to_target_dict(self) -> Dict[str, Any]:
```

Comment:
Should we also add a `python_adapter_type` here?

Reply:
Can you elaborate?

Reply:
`to_target_dict` function returns a dictionary where there is a `type` key pointing to the type of the DB adapter. I was wondering whether it makes sense to add another key called `python_adapter_type` that would point to the type of python credentials (if there is any)